### PR TITLE
feat: Align long user names

### PR DESCRIPTION
**Pull request by dominik003**

### CODE REVIEW

File: frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html [hunk 1]
```diff
@@ -26,8 +25,8 @@ <h2>Project members</h2>
       </div>
     </div>
   </div>
-  <mat-card>
-    <mat-form-field id="search" appearance="outline">
+  <mat-card class="w-[400px] max-w-[85vh] !pb-0.5 pr-0">
```
Comment (MoritzWeber0) [inline frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html:28]:
```suggestion
  <mat-card class="w-[400px] max-w-[85vw] !pb-0.5 pr-0">
```
The max-width has to depend on the width, not the height. The current change break the mobile view.

File: frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html [hunk 2]
```diff
@@ -50,18 +51,20 @@ <h2>Project members</h2>
...
-            >
+            <div class="flex items-center mr-4 ml-0">
+              <mat-icon class="text-3xl !h-8 !w-8 my-auto" mat-list-icon
+                >account_circle</mat-icon
+              >
+            </div>
             <div>
-              <div class="username">{{ user.user.name }}</div>
-              <div class="userrole">
+              <div class="text-xl break-all">{{ user.user.name }}</div>
```
Comment (MoritzWeber0) [inline frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html:60]:
```suggestion
              <div class="text-[1.25em] break-all">{{ user.user.name }}</div>
```

The username is now way larger than before, taking a lot more space.

File: frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html [hunk 3]
```diff
@@ -37,11 +36,13 @@ <h2>Project members</h2>
       />
       <mat-icon matSuffix>search</mat-icon>
     </mat-form-field>
-    <div class="scrollable">
+    <div class="verflow-y-scroll max-h-[50vh]">
```
Comment (MoritzWeber0) [inline frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html:39]: The overflow scrolling doesn't work in the mobile view. 

image

**Review by MoritzWeber0 — CHANGES_REQUESTED** [review]

File: frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html (hunk 1 above)
Comment (dominik003) [reply frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html:28]: Good catch!

File: frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html (hunk 3 above)
Comment (dominik003) [reply frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html:39]: The overflow also didn't work in the normal view. But this was just a typo, i.e., there was just an `o` missing in front

**Review by MoritzWeber0 — APPROVED** [review]

**Pull request merged**